test/test_symmbasis.jl

```diff
@@ -73,7 +73,58 @@ for L = 0 : 3
       println()
 end
 
-##
+#---
+@info("SymmetricBasis construction and evaluation: Spherical Matrix")
+
+for L1 = 0:3
+   for L2 = 0:3
+
+      if L1 == 0
```

Comment:
this is extremely ugly, why not define a function that gives you the symbol associated with each `L`? e.g., something along the lines of
```julia
const __L2syms = [:s, :p, :d, :f, :g, :h, :i, :k]
const __syms2L = Dict( [sym => L-1 for (L, sym) in enumerate(__L2syms)]... )
get_orbsym(L::Integer)  = __Lsyms[L+1]
get_L(sym::Union{AbstractString, Char, Symbol}) = __syms2L[ Symbol(sym) ]
```
should go somewhere inside the ACE module, not entirely sure yet where.

Reply:
and don't worry with the test numbers please